### PR TITLE
updated geoips_clavrx test_scrips to adhere to new schema changes to geoips

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#211

# Testing Instructions
To test the changes made in this PR, run the command ./geoips_clavrx/tests/test_all.sh

# Summary
# Refactoring Updates
=================

* updated all .sh files in the directory geoips_clavrx/tests/scripts to adhere to new schema.
* NOTE: @mindyls @jsolbrig  The test $GEOIPS_PACKAGES_DIR/geoips/tests/utils/check_code.sh all `dirname $0`/../" \ in "test_all.sh" still fails, however it says it passes in the log file associated with the test. Cannot figure this one out, nor am I sure if it actually fails. 

# Output

See NRLMMD-GEOIPS/geoips#211 - all tests pass

(geoips) [evrose54@winter geoips_clavrx]$ ./tests/test_all.sh 
Tue May 2 12:21:27 MDT 2023 Running pre, log: /home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log

Tue May 2 12:21:27 MDT 2023  Running check_code.sh
/home/evrose54/geoips/geoips_packages/geoips/tests/utils/check_code.sh all ./tests/../
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_check_code.sh__all_._tests_.._.log
        Return: 1

Tue May 2 12:21:28 MDT 2023  Running abi_cldFraction.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/abi_cldFraction.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_abi_cldFraction.sh.log
        Return: 0

Tue May 2 12:22:13 MDT 2023  Running ahi_cldFraction.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldFraction.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldFraction.sh.log
        Return: 0

Tue May 2 12:22:58 MDT 2023  Running ahi_cldHeightBase.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldHeightBase.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldHeightBase.sh.log
        Return: 0

Tue May 2 12:23:44 MDT 2023  Running ahi_cldHeight.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldHeight.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldHeight.sh.log
        Return: 0

Tue May 2 12:24:31 MDT 2023  Running ahi_cldHeightTop.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldHeightTop.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldHeightTop.sh.log
        Return: 0

Tue May 2 12:24:56 MDT 2023  Running ahi_cldMask.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldMask.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldMask.sh.log
        Return: 0

Tue May 2 12:25:41 MDT 2023  Running ahi_cldOpd.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldOpd.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldOpd.sh.log
        Return: 0

Tue May 2 12:26:26 MDT 2023  Running ahi_cldPhase.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldPhase.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldPhase.sh.log
        Return: 0

Tue May 2 12:27:11 MDT 2023  Running ahi_cldReff.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldReff.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldReff.sh.log
        Return: 0

Tue May 2 12:27:57 MDT 2023  Running ahi_cldTemp.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldTemp.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldTemp.sh.log
        Return: 0

Tue May 2 12:28:43 MDT 2023  Running ahi_cldType.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_cldType.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_cldType.sh.log
        Return: 0

Tue May 2 12:29:28 MDT 2023  Running ahi_Temp11p0.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_Temp11p0.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_Temp11p0.sh.log
        Return: 0

Tue May 2 12:30:15 MDT 2023  Running ahi_Temp3p75.sh
/home/evrose54/geoips/geoips_packages/geoips_clavrx/tests/scripts/ahi_Temp3p75.sh
/home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log_ahi_Temp3p75.sh.log
        Return: 0

Tue May 2 12:31:01 MDT 2023  Running post, final results in /home/evrose54/geoips/test_data/logs/20230502/20230502.182127_geoips_clavrx/test_all_geoips_clavrx.log

Package: geoips_clavrx
Total run time: 574 seconds
Number data types run: 14
Number data types failed: 1

### CODE REVIEW

File: tests/test_all.sh
Comment (mindyls) [inline tests/test_all.sh:39]: I added "test_interfaces" to the test script, for good measure.

**Review by mindyls — CHANGES_REQUESTED** [review]
No changes required, I just want to make sure we wait until geoips repo is fully updated before we merge these PRs.  It is good to go as soon as geoips is done.

**Review by mindyls — APPROVED** [review]
@jsolbrig This looks good to me.  Feel free to merge and tag 1.10.0a2 if you are happy with these changes.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**
